app/services/audit_results.rb

```diff
@@ -14,96 +14,97 @@
 #   result2 = {response_code => msg}
 class AuditResults
 
-  INVALID_ARGUMENTS = :invalid_arguments
-  VERSION_MATCHES = :version_matches
   ACTUAL_VERS_GT_DB_OBJ = :actual_vers_gt_db_obj
```

Comment:
I was going batty trying to match lists, so I alphabetized them all as a distinct commit.